docs/wustl_htcf.md

```diff
@@ -0,0 +1,21 @@
+# nf-core/configs: WUSTL High Throughput Computing Facility Configuration
+
+Forked from https://github.com/nf-core/configs/blob/master/docs/prince.md
+
+## nf-core pipelines that use this repo
+
+All nf-core pipelines that use this config repo (which is most), can be run on the HTCF. **Before** running a pipeline for the first time, go into an interactive slurm session on a compute node (`srun --pty --time=02:00:00 -c 2`), as the docker image for the pipeline will need to be pulled and converted.
+
+Now, run the pipeline of your choice with `-profile wustl_htcf`. This will download and launch wustl_htcf.config which has been pre-configured with a setup suitable for the HTCF cluster. Using this profile, a docker image containing all of the required software will be downloaded, and converted to a singularity image before execution of the pipeline. This step **takes time**!!
```

Comment:
```suggestion
Now, run the pipeline of your choice with `-profile wustl_htcf`. This will download and launch `wustl_htcf.config` which has been pre-configured with a setup suitable for the HTCF cluster. Using this profile, a docker image containing all of the required software will be downloaded, and converted to a singularity image before execution of the pipeline. This step **takes time**!!
```